### DISCUSSION TITLE
(ruby.portable) Migrate package

### DISCUSSION THREAD

**Discussion by TheCakeIsNaOH** [chocolatey-community/chocolatey-packages#1756] [Migrations]
The `ruby.portable` package is out of date and is currently broken due to the shutdown of bintray. 
https://community.chocolatey.org/packages/ruby.portable

The current maintainer, @ferventcoder, is not actively maintaining packages on CCR at the moment. This package is used by the ruby alternate source for the Chocolatey CLI install command, so it is important that it is working, thus the suggestion by @AdmiringWorm to bring it into this repository.

It should be able to be updatable by AU with a similar script to the one for the `ruby` package.

As a part of this migration, the `ruby` package should probably be converted to a virtual package, and a `ruby.install` package should be created.

I'm not sure who should be responsible for this package, perhaps someone would be able to volunteer?

**Comment by AdmiringWorm** [comment]
Thank you for opening the discussion as I had requested.

While we don't know exactly how will be responsible for the package yet, I assume it will be 1 or more people in the Core team (most likely a Repository Member) as this can be considered a Core package since it is being used by Chocolatey CLI itself for one of the custom sources.

I'll get an issue created based on this discussion with a checklist of what needs to be done.

**Comment by AdmiringWorm** [comment]
I am closing this discussion now that the package has been migrated.
